core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MultiregionalBucketMigration.scala

```diff
@@ -322,6 +319,15 @@ trait MultiregionalBucketMigrationHistory extends DriverComponent with RawSqlQue
           )
         }
 
+        _ <- MonadThrow[ReadWriteAction].raiseWhen(!location.getOrElse("").equals("US")) {
```

Comment:
This check needs to take place only when scheduling a first migration or we would prevent restarts for workspace's that had already had their final workspace bucket created in a single region.